### PR TITLE
a11y(chips): Add accessibility demo page for chips

### PR DESCRIPTION
**Pull request by tinayuangao**

### CODE REVIEW

**Comment by jelbourn** [comment]
@tinayuangao does this need to be rebased on another PR?

**Comment by tinayuangao** [comment]
@jelbourn Yes. The base PR #5748 is not merged yet

File: src/demo-app/a11y/chips/chips-a11y.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<section>
+  <h2>Basic chips</h2>
+  <md-chip-list>
+    <md-chip>Chip 1</md-chip>
+    <md-chip>Chip 2</md-chip>
+    <md-chip>Chip 3</md-chip>
+  </md-chip-list>
```
Comment (jelbourn) [inline src/demo-app/a11y/chips/chips-a11y.html]: Same comment on more "real world" content here

File: src/demo-app/a11y/chips/chips-a11y.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+</section>
+
+<section>
+  <h2>Colored chips</h2>
+  <md-chip-list>
+    <md-chip color="primary" selected="true">Primary</md-chip>
+    <md-chip color="accent" selected="true">Accent</md-chip>
+    <md-chip color="warn" selected="true">Warn</md-chip>
+  </md-chip-list>
+</section>
```
Comment (jelbourn) [inline src/demo-app/a11y/chips/chips-a11y.html]: Maybe add a comment here that this example is good to have for contrast-radio checking

**Comment by tinayuangao** [comment]
Changed to real world examples. Please take another look. Thanks!

File: src/demo-app/a11y/chips/chips-a11y.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+<section>
+  <h2>Removable chips in input container</h2>
+
+  <md-input-container>
+    <md-chip-list mdPrefix #chipList>
+      <md-chip *ngFor="let person of people" [color]="color">
+        {{person.name}}
+      </md-chip>
+    </md-chip-list>
+    <input mdInput placeholder="New Contributor..."
```
Comment (jelbourn) [inline src/demo-app/a11y/chips/chips-a11y.html]: Should add an `aria-label` for the input

**Pull request merged**